### DISCUSSION TITLE
What is the best way to load metaplugins in turbo mode?

### DISCUSSION THREAD

**Discussion by aidan-gibson** [z-shell/zi#256] [Q&A]
So this is valid:
```
zi light-mode for z-shell/z-a-meta-plugins \
  @annexes+ @console-tools @ext-git @fuzzy @py-utils @rust-utils @sharkdp @z-shell @zsh-users+fast @zunit @zsh-users
```
And this is valid:
```
zi light z-shell/z-a-meta-plugins
zi ice wait
zi light @annexes+
```

How can I essentially do this?
```
zi-turbo '1a' for z-shell/z-a-meta-plugins \
  @annexes+ @console-tools @ext-git @fuzzy @py-utils @rust-utils @sharkdp @z-shell @zsh-users+fast @zunit @zsh-users
```

I'm aware this is probably a really stupid question, would super appreciate the help!

**Comment by ss-o** [comment]
It is important that plugins do not overwrite each other (settings conflict) and are loaded in the correct order, in other words, check each plugin group behavior and try to match the most compatible, then arrange them in the best order.
